openlibrary/core/models.py

```diff
@@ -644,45 +644,48 @@ def resolve_redirects_bulk(
         cls,
         batch_size=1000,
         start_offset=0,
-        limit=1000,
         grace_period_days=7,
```

Comment:
It would also be nice if grace_period could be specified as an int or a datetime (which is basically a `start_date`